danielMunteanu/src/main/java/Generics/GenericLibrary.java

```diff
@@ -0,0 +1,35 @@
+package Generics;
+
+import java.util.List;
+
+public class GenericLibrary<T extends LibraryItem> {
```

Comment:
add please more interesting methods
like printByTipe(type) and printByYear(year)/printByYears(from/to)

Reply:
done